app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,159 @@
+  box-shadow: inset $horizontal $vertical $blur $color;
 }
 
 @mixin gradient($color-from, $color-to) {
-    background-image: -webkit-gradient(linear, 0 0, 0 100%, from($color-from),to($color-to));
-    background-image: -webkit-linear-gradient($color-from, $color-to);
-    background-image: -moz-linear-gradient(top, $color-from, $color-to);
-    background-image: -ms-linear-gradient($color-from, $color-to);
-    background-image: -o-linear-gradient($color-from, $color-to);
-    background-image: linear-gradient($color-from, $color-to);
-    background-repeat: no-repeat
+  background-image: -webkit-gradient(linear, 0 0, 0 100%, from($color-from),to($color-to));
+  background-image: -webkit-linear-gradient($color-from, $color-to);
+  background-image: -moz-linear-gradient(top, $color-from, $color-to);
+  background-image: -ms-linear-gradient($color-from, $color-to);
+  background-image: -o-linear-gradient($color-from, $color-to);
+  background-image: linear-gradient($color-from, $color-to);
+  background-repeat: no-repeat
 }
 
 @mixin display-flex() {
-    display: -ms-flexbox;
-    display: -webkit-flex;
-    display: -moz-flex;
-    display: -ms-flex;
-    display: flex;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: -moz-flex;
+  display: -ms-flex;
+  display: flex;
 }
 @mixin flex-flow($flex-dir: row, $flex-wrap: nowrap) {
-    -webkit-flex-flow: $flex-dir $flex-wrap;
-    -moz-flex-flow: $flex-dir $flex-wrap;
-    -ms-flex-flow: $flex-dir $flex-wrap;
-    flex-flow: $flex-dir $flex-wrap;
+  -webkit-flex-flow: $flex-dir $flex-wrap;
+  -moz-flex-flow: $flex-dir $flex-wrap;
+  -ms-flex-flow: $flex-dir $flex-wrap;
+  flex-flow: $flex-dir $flex-wrap;
 }
 
 @mixin justify-content($arg: flex-start) {
-    -webkit-justify-content: $arg;
-    -moz-justify-content: $arg;
-    -ms-justify-content: $arg;
-    justify-content: $arg;
+  -webkit-justify-content: $arg;
+  -moz-justify-content: $arg;
+  -ms-justify-content: $arg;
+  justify-content: $arg;
 }
 
 @mixin flex($arg: 1) {
-    -webkit-flex: $arg;
-    -moz-flex: $arg;
-    -ms-flex: $arg;
-    flex: $arg;
+  -webkit-flex: $arg;
+  -moz-flex: $arg;
+  -ms-flex: $arg;
+  flex: $arg;
 }
 
 @mixin transition-all($time) {
-    -webkit-transition: all $time ease;
-    -moz-transition: all $time ease;
-    o-transition: all $time ease;
-    transition: all $time ease;
-
+  -webkit-transition: all $time ease;
+  -moz-transition: all $time ease;
+  -o-transition: all $time ease;
+  transition: all $time ease;
 }
 
 a.button {
-    &:hover {
-       color: $medium-green;
-    }
+  &:hover {
+    color: $medium-green;
+  }
 }
 
 a.transparent-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $white;
-    background-color: transparent;
-    color: $white;
-    &:hover {
-        color: $white;
-        background-color: rgba(0, 0, 0, 0.4);
-    }
+  @extend .button, .large;
+  @include transition-all(100ms);
+  border-radius: $global-radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, font-weight